src/test/java/org/cactoos/func/RunnableOfTest.java

```diff
@@ -48,18 +48,18 @@ public void convertsFuncIntoRunnable() {
                 }
             ),
             new MatcherOf<Runnable>(
-                input -> {
-                    input.run();
+                input1 -> {
+                    input1.run();
```

Comment:
@andreoss This name change `input` to `input1` was intentional? If yes, please, add a better name than `input1`.

Reply:
@fabriciofx Done